### PR TITLE
Fix formula

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by bqi343** [comment]
@brebenelmihnea Can you follow the instructions here?

https://github.com/cpinitiative/usaco-guide/pull/4563#issuecomment-2211048860

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
btw i don't see u changing this to c_n-1 as stated in the issue

File: content/4_Gold/Combinatorics.mdx [hunk 1]
```diff
@@ -829,7 +829,7 @@ $$
 
 They also have the recurrence formula
 $$
-C_n= \sum^{n}_{i=1}{C_i \cdot C_{n-i}} \,\,\, \text{ for } n>0 \\
+C_{n+1}= \sum^{n}_{i=0}{C_i \cdot C_{n-i}} \,\,\, \text{ for } n>0 \\
```
Comment (bqi343) [inline content/4_Gold/Combinatorics.mdx:832]:
```suggestion
C_{n+1}= \sum^{n}_{i=0}{C_i \cdot C_{n-i}} \,\,\, \text{ for } n\ge0 \\
```

Comment (brebenelmihnea) [reply content/4_Gold/Combinatorics.mdx:832]: ok

**Pull request merged**